src/modules/pages/LoginPage/LoginPage.js

```diff
@@ -1,16 +1,7 @@
 import React from 'react';
 
-import Section from '../../Layout/Section/Section';
-import Title from '../../Layout/Title/Title';
 import LoginContainer from './loginContainer';
 
-const LoginPage = () => (
-    <div>
-        <Section>
-            <Title>Login page</Title>
-        </Section>
-        <LoginContainer />
-    </div>
-);
+const LoginPage = () => <LoginContainer />;
```

Comment:
Sure, it was possible to omit this file by passing straight to the root, but I left it just for consistency and for the case if something except of the form will be needed on this page